docs/source/release-notes/v86-k2.md

```diff
@@ -0,0 +1,104 @@
+# K2.0 Release Notes
+
+This version of Project Athena is compatible with the v0.86.0 protocol.
+
+
+## Interface
+
+The numbers at the end of each item are the PR numbers in the Project Athena [GitHub repo](https://github.com/kasenvr/project-athena/pulls?utf8=%E2%9C%93&q=milestone%3Av0860-k2).
+
+#### General
+
+* Settings > Entity Script / QML Whitelist added. Loading third-party QML now works correctly, provided that the file is added to the whitelist. (#1, #6, #7, #16, #65, #112)
+* The ability for the Metaverse API (Grid Server) URL to be changed by script via settings has been added. No UI toggle has been added yet. (#33)
+* Added full desktop chat UI with local, domain, and inter-domain capabilities. Not in a VR usable state yet. (#12, #13, #69, #88, #107, #111, #135, #170)
+* Serverless JSON domains can now be loaded over HTTP. This means you can create a domain, export it to JSON, then upload it to a webserver to be visited in the Interface. (#160)
+* Beta default scripts feature added where if you check Developer > Scripting > Enable Beta Default Scripts then you get the latest versions for some scripts via their respective online deployments. (#135, #192)
+* Users can now set thumbnails that show in the Avatar app. (#145)
+* Add SL-like camera inspect function (hold Alt key) as default functionality. (#64, #188)
+* Add clicking/triggering on an avatar shows a temporary nametag. (#179)
+* Add an "Explore" app as a temporary go-to, along with associated "Explore" menu in Interface. These will be replaced by a modern implementation upon or after the Metaverse API (Grid Server) completion. (#190, #199, #205)
+* Add a "More" app that lists and installs community-made scripts and apps directly from a [GitHub repo](https://github.com/kasenvr/community-apps).
+* This takes longer than you might expect: Tons of revisions to brand/debrand, update links, docs, rehost, etc. in the Interface by various contributors. Thank you! (#9, #11, #19, #23, #34, #37, #38, #40, #44, #47, #50, #58, #70, #71, #75, #85, #89, #91, #99, #100, #105, #116, #177, #219)
+
+#### Graphics
+
+* Procedural shaders on mesh (Avatars, Models) support added. (#???)
+* Procedural vertex shader support added. (#15)
+* Procedural shaders no longer require an environment variable to be activated. Instead, use Developer > Render > Enable Procedural Materials. (#8)
+* Add support for GLTF alphaModes. (#15)
+* Fix opacityMapModes for GLTF. (#15)
+</br><img src="https://user-images.githubusercontent.com/26217480/70808925-be179700-1d8e-11ea-967a-9d1c84eff57c.PNG" width="60%"/>
```

Comment:
Let's move the PNG into the api-docs-sphinx repo, E.g., into a new directory, docs/source/release-notes/_images (matches other directories' naming).

Reply:
All done.